packages/bpk-docs/src/components/neo/AlternatingPageContent/AlternatingPageContent.js

```diff
@@ -25,14 +25,19 @@ import STYLES from './AlternatingPageContent.scss';
 const getClassName = cssModules(STYLES);
 
 type Props = {
+  invert: boolean,
   sections: Array<?Node>,
 };
 
 const AlternatingPageContent = (props: Props) => (
   <section className={getClassName('bpkdocs-alternating-content')}>
     {React.Children.toArray(
       props.sections.filter(x => x).map((section, i) => {
-        const isEven = i % 2 === 0;
+        let index = i;
+        if (props.invert) {
+          index += 1;
+        }
+        const isEven = index % 2 === 0;
```

Comment:
Being really picky, but bumping the index implies that we're skipping a section. Maybe put this logic inline instead?
`const isEven = index + props.invert ? 1 : 0 % 2 === 0;`

Then again, maybe not

Reply:
I've refactored the logic a bit to make it hopefully clearer.